docs/GettingStartedDocs/GettingStarted.Windows.md

```diff
@@ -103,6 +103,20 @@ C:\> where ocaml
 C:\Program Files\ocpwin64\4.02.1+ocp1-msvc64-20160113\bin\ocaml.exe
 ```
 
+Configuration script
```

Comment:
This section needs to be moved up before we describe individually installing each of the components, and should be presented as an alternative. For example, move this under the Prerequisites header at the start of the doc:

> Prerequisites
> -------------
>
> The following are prerequisites for building and running Open Enclave on Windows.
> 
> ...
> 
> To deploy all the the prerequisites for building Open Enclave, you can run powershell script `scripts/oe-env-prepare.ps1`:
> ```powershell
> cd scripts
> .\oe-env-prepare.ps1
> ```
> 
> To deploy each prerequisite individually, refer to the sections below.